app/index.html

```diff
@@ -0,0 +1,91 @@
+<html>
+<head>
+    <title>Bridge Researcher Portal</title>
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/semantic-ui/1.12.0/semantic.min.css"/>
+</head>
+<body class="HolyGrail">
+    <header>
+        <h2>Bridge Researcher Portal</h2>
+        <a href="#signOut">Sign Out</a>
+    </header>
+    <div class="HolyGrail-body">
+        <main class="HolyGrail-content" id="page-container">
+            <div data-bind="component: mainPage"></div>
+        </main>
+        <nav class="HolyGrail-nav">
+            <div class="ui left vertical menu">
+                <a href="#home" class="item">
+                    <i class="home icon"></i>Home
+                </a>
+                <a href="#study" class="item">
+                    <i class="lab icon"></i>Study
+                </a>
+            </div>
+        </nav>
+        <!--
+        <aside class="HolyGrail-ads">
+        </aside>
+        -->
+    </div>
+    <footer>
+        <p>&copy; 2015 Sage Bionetworks. <span id="sessionToken"></span></p>
+    </footer>
+
+    <div id="authModal" class="ui small modal">
```

Comment:
There's going to be a lot of modals, we don't want to load them all at once... though this is application-scoped.